main.py

```diff
@@ -25,93 +25,32 @@ def __init__(self):
 
         # Used when moving two frames
         self.selected_frame = None
+        self.fullscreen = False
 
     def setup_ui(self):
         """Loads the main.ui file and sets up the window and grid."""
-        self.ui = uic.loadUi("ui/main.ui")
+        self.ui = uic.loadUi("ui/main.ui", self)
```

Comment:
Previously, `self` was not passed in upon building the UI, so Qt built a "new" application (although it looked the same way). So when issuing showFullScreen() on the application the class "Application" would become fullscreen, but the containing videos inside would not (since they didn't belong to that window). This also has to be done to avoid memory leaks between PyQt and Qt itself.